test/external/druidExternal.mocha.coffee

```diff
@@ -89,6 +89,24 @@ describe "DruidExternal", ->
         apply(Volatile,$_sd_0:NUMBER.subtract($_sd_1:NUMBER))
         """)
 
+    it "breaks up correctly in absolute cases", ->
+      ex = ply()
```

Comment:
not sure what this is testing, there is nothing here to break up. maybe put the .absolute call in a more complex expression?